.buildkite/pull-requests.json

```
@@ -27,7 +27,7 @@
       "build_on_comment": true,
       "trigger_comment_regex": "^(?:(?:buildkite\\W+)?(?:build|test)\\W+(?:this|it))|^/test$",
       "always_trigger_comment_regex": "^(?:(?:buildkite\\W+)?(?:build|test)\\W+(?:this|it))|^/test$",
-      "fail_on_not_mergeable": true,
-   }
-   ]
+      "fail_on_not_mergeable": true
```

let's use the same settings as in the other pipeline
```suggestion
      "skip_ci_labels": [ ],
      "skip_target_branches": [ ],
      "skip_ci_on_only_changed": [ ],
      "always_require_ci_on_changed": [ ]
```